openpype/hosts/maya/plugins/publish/validate_yeti_renderscript_callbacks.py

```diff
@@ -48,6 +48,10 @@ def get_invalid(cls, instance):
 
         yeti_loaded = cmds.pluginInfo("pgYetiMaya", query=True, loaded=True)
 
+        if not yeti_loaded or not cmds.ls(type="pgYetiMaya"):
+            cls.log.info("No yeti nodes found.")
+            return False
+
```

Comment:
```suggestion
        if not yeti_loaded and not cmds.ls(type="pgYetiMaya"):
            # The yeti plug-in is available and loaded so at
            # this point we don't really care whether the scene
            # has any yeti callback set or not since if the callback
            # is there it wouldn't error and if it weren't then
            # nothing happens.
            cls.log.info(
                "Yeti is loaded but no yeti nodes were found. "
                "Callback validation skipped.."
            )
            return False

```